scripts/download_data.py

```diff
@@ -280,14 +293,36 @@ def downloadFastas(self):
                 'display': 'fasta',
                 'range': '{}-{}'.format(0, maxPos)})
             url = '{}%26{}'.format(path, args)
+            tempFileName = '{}.fa.temp'.format(chromosome)
             fileName = '{}.fa'.format(chromosome)
-            downloader = utils.HttpFileDownloader(url, fileName)
+            downloader = utils.HttpFileDownloader(url, tempFileName)
             downloader.download()
+            # We need to replace the header on the downloaded FASTA
+            with open(tempFileName, "r") as inFasta,\
```

Comment:
ooo I didn't know one could have multiple clauses in a with statement like this (but a puppy died!)